attributes/default.rb

```diff
@@ -32,15 +32,13 @@
 default['datadog']['agent_major_version'] = nil # nil to autodetect based on 'agent_version'
 
 # Agent Version
-# Default of `nil` will install latest version. On Windows, this will also upgrade to latest
-# This attribute accepts either a `string` or `hash` with the key as platform_name and value of package version
-# In the case of fedora and amazon linux, use platform_name of rhel
+# Default of `nil` will install latest version. On Windows, this will also upgrade to latest.
+# This attribute takes a `string` with a version number. For compatibiliy reasons it can also take a
+# hash with the platform_name as key and the version as value.
+# Starting with version 4.1.0, you no longer need to pass the "1:" prefix and "-1" suffix in versions
+# for Debian-based and SUSE.
 # Example:
-# default['datadog']['agent_version'] = {
-#  'rhel' => '5.9.0-1',
```

Comment:
What about RHEL? We used to add the `-1` revision in the version name, is that not needed anymore? Or do we need to add another case for it?

Reply:
I'll test.

Reply:
On CentOS 6 it seems to work without the added `-1`.

Reply:
Nice. I wonder why we added that `-1` in all tools where we install RHEL though 🤔

Reply:
Works on Fedora 21 as well, I think the `-1` in the example was just a typo.

Reply:
In what other tool were we adding `-1`?

Reply:
It seems to work without the `-1` everywhere I tried.